cryptoki/src/mechanism/mod.rs

```diff
@@ -30,11 +30,17 @@ pub struct MechanismType {
 /// mechanism code is 0x80000170 with no parameters 
 pub const CKM_AES_KEY_WRAP_THALES: CK_MECHANISM_TYPE = 0x80000170;
 
+/// Thales has their own defined Key Wrap mechanism which is compatible with PKCS11 standard mechanism AES_KEY_WRAP_PAD
+/// mechanism code is 0x80000171 with no parameters 
+pub const CKM_AES_KEY_WRAP_PAD_THALES: CK_MECHANISM_TYPE = 0x80000170;
```

Comment:
Value is a duplicate of `CKM_AES_KEY_WRAP_THALES`
```suggestion
pub const CKM_AES_KEY_WRAP_PAD_THALES: CK_MECHANISM_TYPE = 0x80000171;
```

Reply:
Oops thanks for spotting this typo